packages/suite/src/support/rawMessages.ts

```diff
@@ -0,0 +1,6496 @@
+const messages = {
+    TR_3RD_PARTY_WALLETS: {
+        defaultMessage: 'Supported by external wallets',
+        id: 'TR_3RD_PARTY_WALLETS',
+    },
+    TR_3RD_PARTY_WALLETS_DESC: {
+        defaultMessage:
+            'These coins are supported by Trezor firmware but only in external wallets.',
+        id: 'TR_3RD_PARTY_WALLETS_DESC',
+    },
+    TR_404_DESCRIPTION: {
+        defaultMessage: 'Whoops-a-daisy..! Our apologies. Looks like a wrong URL or broken link.',
+        id: 'TR_404_DESCRIPTION',
+    },
+    TR_404_GO_TO_DASHBOARD: {
+        defaultMessage: 'Go to Dashboard',
+        id: 'TR_404_GO_TO_DASHBOARD',
+    },
+    TR_404_TITLE: {
+        defaultMessage: 'Error 404: Link not found',
+        id: 'TR_404_TITLE',
+    },
+    TR_ACCESS_HIDDEN_WALLET: {
+        defaultMessage: 'Access Hidden wallet',
+        id: 'TR_ACCESS_HIDDEN_WALLET',
+    },
+    TR_WALLET_SELECTION_ACCESS_HIDDEN_WALLET: {
+        defaultMessage: 'Access Hidden wallet\n',
```

Comment:
There is `\n` at the end of many messages. I believe, that it is just unintentional extra enter in Crowdin message. What about to filter them out in the script?

Reply:
But are we sure that this is not intentional? It might have been intention of the translator.. maybe?

Reply:
But what would be the intention? To change the layout and add some extra space after? It doesn't work like that, in most cases there is no visible change. 

I believe, that `\n` inside the text is ok, but not at the end.

Reply:
checkout here 214e416b2a4b805b317354363ad6271249e69220